FourthTask/figures/main/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/tests"
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types"
+)
+
+func main() {
+	if err := tests.Errors(); err != nil { //Programm will start work after tests check
```

Comment:
it's not the best practice to mix test code and logic.
Usually it solved with CI tools or (sigh...) Makefiles.